tools/prep_cc4_code.py

```diff
@@ -259,28 +302,18 @@ def add_type_logos(self, content, lic_type):
             filename = UpdateLicenseCode.image_map[lic_attr]["file"]
             alt_text = UpdateLicenseCode.image_map[lic_attr]["alt_text"]
             image_tag = (
-                '<img src="/images/deed/svg/'
-                + filename
-                + '" alt="'
-                + alt_text
-                + '"/>'
+                f'<img src="/images/deed/svg/{filename}" alt="{alt_text}"/>'
             )
             lic_images += (
-                '<span class="cc-icon-'
-                + lic_attr
-                + '">'
-                + image_tag
-                + "</span>"
+                f'<span class="cc-icon-{lic_attr}">{image_tag}</span>'
             )
         cc_logo_section = re.search(
             '<div id="cc-logo">.*?</div>', content, re.DOTALL
         ).group()
         new_cc_logo_section = (
-            '<div id="cc-logo">'
-            + '<span class="cc-icon-logo">'
-            + '<img src="/images/deed/svg/cc_white.svg" alt="CC"/></span>'
-            + lic_images
-            + "</div>"
+            '<div id="cc-logo"><span class="cc-icon-logo">'
+            '<img src="/images/deed/svg/cc_white.svg" alt="CC"/></span>'
```

Comment:
Could you change the `alt` attribute to be `"CC Logo"`? I think that would be more helpful in this case.